migrations/20190605091046-Table-Keywords.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function(db) {
+  return  db.createTable('keyword', {
+    id: {
+      type: 'int', primaryKey: true, autoIncrement: true, notNull: true,
+    },
+    name: { type: 'char', length: 80, notNull: true },
+    importance: { type: 'int', notNull: true },
+  })
+  .then(
+    function(result){
```

Comment:
creating tables and filling tables should not be part of the same file . one day you will want to create the db without seeding it